scripts/templates/nxiq-operator-certified.clusterserviceversion.yaml

```diff
@@ -212,6 +218,7 @@ spec:
     | `iq.applicationPort` | Port of the application connector. Must match the value in the `configYaml` property | `8070`            |
     | `iq.adminPort`       | Port of the application connector. Must match the value in the `configYaml` property | `8071`            |
     | `iq.memory`          | The amount of RAM to allocate                                | `1Gi`             |
+    | `iq.env`             | Customize the server environment, including `JAVA_OPTS`      | See example YAML show when ceating a NexusIQ. |
```

Comment:
these docs are displayed in the description of the operator in openshift console.